xbmc/network/httprequesthandler/IHTTPRequestHandler.h

```diff
@@ -22,6 +22,14 @@
 #include <sys/socket.h>
 #include <sys/types.h>
 
+#include <microhttpd.h>
```

Comment:
This include already exist at the correct place a few lines above.

Reply:
Thanks!